hack/tools/create_kind.sh

```diff
@@ -83,6 +102,15 @@ case "$op" in
         registry_delete || true
         certs_delete || true
         kind_delete || true
+        kind_delete_control || true
+        ;;
+    multi)
```

Comment:
I wonder if we would want to create/delete one cluster independently

Reply:
I used parameters for create and delete now. Does that also solve this comment?

Reply:
See https://github.com/IBM/the-mesh-for-data/pull/142#pullrequestreview-534300178 to understand what I meant, but we are good for now.